Websites/perf.webkit.org/public/include/uploaded-file-helpers.php

```diff
@@ -160,16 +163,17 @@ function delete_file($db, $file_row)
     return TRUE;
 }
 
-function prune_old_files($db, $size_needed, $remote_user)
+function prune_old_files($db, $size_needed, $remote_user, $file_deletion_cutoff)
 {
-    $user_filter = $remote_user ? 'AND file_author = $1' : 'AND file_author IS NULL';
-    $params = $remote_user ? array($remote_user) : array();
+    $file_filters = 'AND extract(epoch from file_created_at at time zone \'utc\') <= $1';
+    $file_filters .= ($remote_user ? ' AND file_author = $2' : ' AND file_author IS NULL');
+    $params = $remote_user? array($file_deletion_cutoff, $remote_user) : array($file_deletion_cutoff);
 
-    // 1. Delete old build products created for a patch not associated with any pending or in-progress builds.
-    $build_product_query = $db->query("SELECT file_id, file_extension, file_size FROM uploaded_files, commit_set_items
-        WHERE file_id = commitset_root_file AND commitset_patch_file IS NOT NULL AND file_deleted_at IS NULL
+    // 1. Delete old build products created not associated with any pending or in-progress builds.
```

Comment:
*Delete old build products not associated* or *Delete old build products created for any pending...*